client/src/components/AuthorName.jsx

```diff
@@ -2,25 +2,30 @@ import React from 'react';
 import FollowButton from './FollowButton.jsx';
 import styled from 'styled-components';
 
+const Div = styled.div`
+  display: flex;
+  /* font-size:0; */
```

Comment:
I would remove all the commented/unused styles. & btw, you would definitely need a clean up day for the dead & non-conforming style guide code 😄. 
Also be sure restructure your project files & update the `README `

Reply:
Yes, cleaning up my code is on my list, though it's not a priority right now. Also, re: restructure your project files - for this point, do you have something specific in mind?

Reply:
Essentially using the `server` directory to include the back-end related code.